### PR TITLE
`README.md`: update the `gem install` demo 

### PR DESCRIPTION
**Pull request by soda92**
## What was the end-user or developer problem that led to this PR?

The install instruction on the README is outdated and do not reflect the current status for Windows.

See <https://github.com/sparklemotion/nokogiri.org/pull/67>

## What is your fix for the problem, implemented in this PR?

add install instructions on Windows

## Make sure the following tasks are checked

- [x] Describe the problem / feature
- [ ] Write [tests](https://github.com/rubygems/rubygems/blob/master/bundler/doc/development/PULL_REQUESTS.md#tests) for features and bug fixes
- [ ] Write code to solve the problem
- [x] Make sure you follow the [current code style](https://github.com/rubygems/rubygems/blob/master/doc/bundler/development/PULL_REQUESTS.md#code-formatting) and [write meaningful commit messages without tags](https://github.com/rubygems/rubygems/blob/master/doc/bundler/development/PULL_REQUESTS.md#commit-messages)

### CODE REVIEW

**Comment by soda92** [comment]
related: https://github.com/rubygems/rubygems/pull/8253

**Comment by soda92** [comment]
Also, I wonder if `nokogiri` is really a good example for showing `gem install`.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,18 @@ you read and parse XML in Ruby:
...
+```
+ridk exec pacman -S mingw-w64-ucrt-x86_64-libxslt
+gem install nokogiri --platform=ruby -- --use-system-libraries
+```
+
+[noko-doc]: https://nokogiri.org/tutorials/installing_nokogiri.html#windows
+
+You can also find detailed install instructions for other platforms in the documentation.
+
+When executing `gem install`, RubyGems will download the Nokogiri Gem from RubyGems.org and install it into your Ruby environment.
```
Comment (deivid-rodriguez) [inline README.md:29]: This seems very specific to nokogiri. I think the whole point of this section is to give a basic example of how to install and use a gem, so maybe we should switch to a simpler gem?

**Comment by deivid-rodriguez** [comment]
> Also, I wonder if `nokogiri` is really a good example for showing gem install.

Sorry, I totally missed this comment. As you saw in my comments, I completely agree 👍. Let's use `json` for example.

**Comment by deivid-rodriguez** [comment]
Actually, `json` is probably not a great example either, because since it's available by default anyways without `gem install json`, that may be confusing.

Maybe [tzinfo](https://github.com/tzinfo/tzinfo) or [faraday](https://github.com/lostisland/faraday) (just going through the [top gems list](https://rubygems.org/stats) and trying to find a simple example)?

**Comment by soda92** [comment]
We can just provide instructions on the Ruby site, or provide link to them:
<https://www.ruby-lang.org/en/libraries/#:~:text=A%20few%20more%20words%20about%20RubyGems>

**Comment by deivid-rodriguez** [comment]
Mmmm, I'm not sure we should rely on Ruby website to document our own software. If anything, it should be the other way around (ruby-lang site links to us)?

**Comment by deivid-rodriguez** [comment]
I think [this excerpt](https://lostisland.github.io/faraday/#/getting-started/quick-start?id=usage) from Faraday website essentially shows the same thing we do for nokogiri now: install a gem, use the top level module defined by the gem.

So I'd go with explaining the same thing with `faraday`. Just maybe with `response = Faraday.get('https://rubygems.org')`.

**Comment by soda92** [comment]
Seems fine to me. It can be installed without hassle on Windows

**Review by deivid-rodriguez — APPROVED** [review]
This looks good to me!

**Pull request merged**